ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,42 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.hasRadioSubTextAndLabel 'columns'}}" data-test-radio-input>
         {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-          <div class="column is-narrow is-flex-center has-text-grey has-right-margin-s" data-test-input={{@attr.name}}>
+          <div
+            class="is-flex-center
+              {{if
+                this.hasRadioSubTextAndLabel
+                'has-top-padding-xs has-bottom-padding-s'
+                'column is-narrow has-text-grey has-right-margin-s'
```

Comment:
I think has text grey should be removed here and radio button labels should always be black - this gets us closer to HDS styles
```suggestion
                'column is-narrow has-right-margin-s'
```